02_TrackWeb/src/stats.ts

```
@@ -0,0 +1,246 @@
+    }
+
+    // Aggregate by domain
+    const domainMap = new Map<string, { active: number; total: number; urls: WebsiteTimeEntry[] }>();
+    
+    allEntries.forEach(entry => {
+        if (!domainMap.has(entry.domain)) {
+            domainMap.set(entry.domain, { active: 0, total: 0, urls: [] });
+        }
+        const domainData = domainMap.get(entry.domain);
+        // domainData will always exist after the check and set above
+        domainData!.active += entry.activeSeconds;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)